py3status/modules/gcal.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming calendar events from Google Calendar.
+
+Configuration parameters:
+    agenda_days: show calendar events for number of days (default 1)
+    button_refresh: mouse button to refresh this module (default 2)
+    cache_timeout: refresh interval for this module (default 3600)
+    command: customize gcalcli to dictate what to do with the events
+        (default 'gcalcli agenda --nocolor')
+    format: display format for this module (default '{format_event}')
+    format_event: display format for events
+        (default '[\?if=is_date&color=#ff0 %a %b %d ]
+        [\?if=is_time&color=#0f0 %-I:%M %p ][\?color=#f00 {event}]')
+    format_separator: show separator only if more than one (default ', ')
+    remove: list of strings to remove from the output (default [])
```

Comment:
>If people want colourful output they are welcome to it but forcing it on people feels
>wrong.

Many modules comes forced with `GOOD/BAD/DEGRADED` colors.

I'm not trying to force colors on people. This is to set up something sensible for users to see in default module. I would set up `darkyellow, darkcyan` just like default `gcalcli` output, but I feel that'd go against py3status to set random colors.

Color problem: There are more to things than just `GOOD/BAD/DEGRADED`.

Also, users can see where they can change colors in the format_event. Similar to
sysdata`s `format = [\?color=mem Mem...]`. Now I have to add examples. :-(

>on the 7 AM issue to me this is crazy if the time shows as 7:15 AM then I should also see 7:00 AM this is about consistency which is important.

This is consistency if people quickly add events: "Dinner at 6pm today", "Dentist at 8am today",
"Lunch with Joey at 11am today", "Oil change appointment at 3pm tomorrow", "Basketball
game at 8pm next Thursday"... It all would translates to `X:00 AM/PM` (depending on datetime
styling). Often, we don't need that `:00`. 

>can we just kill this?

I use `remove = [':00']` myself, but wanted an global option so this simple option could be killed and could be useful in other modules.